Project.toml

```
@@ -1,12 +1,13 @@
 name = "RDatasets"
 uuid = "ce6b1742-4840-55fa-b093-852dadbb1d8b"
-version = "0.7.7"
+version = "0.8.0"
```

Probably a good occasion to tag 1.0.0. Clearly the package is stable enough.
```suggestion
version = "1.0.0"
```